stan/math/prim/meta/is_constant.hpp

```diff
@@ -62,5 +62,12 @@ template <typename T>
 struct is_constant<T, require_eigen_t<T>>
     : bool_constant<is_constant<typename std::decay_t<T>::Scalar>::value> {};
 
+template <typename... Types>
+inline constexpr bool is_constant_all_v = is_constant_all<Types...>::value;
+
+template <typename... Types>
+inline constexpr bool is_constant_v
+    = std::conjunction<is_constant<Types>...>::value;
```

Comment:
Actually I think we should start using the std version! We just used the stan math version because we didn't have c++17 available previously


> Also same comment about redundancy - is there a situation where is_constant_all_v would behave differently to is_constant_v here?


Since I made `is_constant_all_v` to accept multiple types there should be no difference. I'm going to put up another PR where throughout the math library I change `is_constant_all` with `is_constant_v`. Which should clean up a lot.